Add recommended badge to Analytics setup checkbox.

abdelmalekkkkk opened the pull request:
## Summary

<!-- Please reference the issue this PR addresses in the following list. -->
Addresses issue:

- #11334

## Relevant technical choices

<!-- Please describe your changes. -->

## PR Author Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 5.2 and PHP 7.4.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [ ] I have added a QA Brief on the issue linked above.
- [ ] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.
- [ ] Ensure there are no unexpected significant changes to file sizes.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

abdelmalekkkkk commented:
The VRT failure is unrelated.

techanvil reviewed (changes requested):
Hey @abdelmalekkkkk, this has been branched off `enhancement/11337-add-exit-setup-to-splash` rather than `develop`, bringing the changes from that branch into this PR and making it harder to review.

Please can you either rebase it to `develop` and update accordingly, or wait until `enhancement/11337-add-exit-setup-to-splash` has been merged before moving this to CR (preferably rebasing/merging `develop` at that point).

Other than that, it's looking good with just one small tweak needed as per my comment.

abdelmalekkkkk commented:
Thanks @techanvil. I was hoping that #11337 would be merged before this was picked up for CR, wasn't the case.

I addressed your comment regarding the padding. Please take another look.
